doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -29,6 +29,15 @@ You can identify and select the appropriate host MCU depending on the IoT use ca
 .. note::
    The asterisk * indicates that the SoC is supported, but the use case is not recommended due to performance or memory constraints.
 
+Firmware patch optimizations
+****************************
+For resource-constrained hosts especially with low internal flash memory, the nRF70 Series firmware patches can significantly contribute to the overall memory footprint.
+The nRF70 Series provides the following firmware patch optimizations to reduce the memory footprint:
+
+* A separate firmware patch is provided for scan only use case that is smaller in size compared to the firmware patch for the :abbr:`STA (Station)` use case.
```

Comment:
```suggestion
* A separate firmware patch variant is provided for applications that only require support for Wi-Fi scan-only operation. This patch variant is significantly smaller in size compared to the full-featured firmware patch variant that supports Wi-Fi :abbr:`STA (Station)` mode.
```

Reply:
Also, I wonder if there could also be a link to the Scan mode (similar to the provided link to the STA mode

Reply:
By `link` are you referring to the abbreviation?